mne/viz/epochs.py

```diff
@@ -69,6 +70,9 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         Figure instance to draw the image to. Figure must contain two axes for
         drawing the single trials and evoked responses. If None a new figure is
         created. Defaults to None.
+    logscale : bool
+        Plots logarithmic value (base 2) of epochs on linear colorscale if True
```

Comment:
Sorry about the nitpick, but it's better if you say:

```If True, transform data to logarithmic scale (base 2) before plotting```

it's more clear to me